rascal-vscode-extension/src/test/vscode-suite/dsl.test.ts

```diff
@@ -137,6 +140,33 @@ parameterizedDescribe(function (errorRecovery: boolean) {
         }
     });
 
+    it("has syntax highlighting in documents without extension", async function () {
+        await bench.executeCommand("workbench.action.files.newUntitledFile");
+        await bench.executeCommand("workbench.action.editor.changeLanguageMode");
+
+        const inputBox = new InputBox();
+        await inputBox.setText("parametric-rascalmpl");
+        await inputBox.confirm();
+
+        const file = "Untitled-1";
+        const editor = await driver.wait(async () => {
+            const result = await ignoreFails(new Workbench().getEditorView().openEditor(file)) as TextEditor;
+            if (result && await ignoreFails(result.getTitle()) === file) {
+                return result;
+            }
+            return undefined;
+        }, Delays.normal, "Could not open file");
+        expect(editor).to.not.be.undefined;
+
+        await editor!.setText(`begin
+  declare
+     a : natural;
+  a := 2;
+end
+`, true);
+        await ide.hasSyntaxHighlighting(editor!, Delays.slow);
```

Comment:
Maybe we should also test that if we add invalid syntax, we get parsing errors reported? so that the parser works, but also the diagnostics propogation.